docs/versioned_docs/version-2.11.0/setup/helm.md

```diff
@@ -11,7 +11,7 @@ This repository contains Helm charts for deploying [ToolJet](https://github.com/
 
 ### From Helm repo
 ```bash
-helm repo add tooljet https://github.com/ToolJet/helm-charts.git
+helm repo add tooljet helm repo add tooljet https://tooljet.github.io/helm-charts
```

Comment:
This is the url we are using - https://github.com/ToolJet/helm-charts. since we maintain separate repo for helm

Reply:
I get the following error if I run the helm repo add
```
$ helm repo add tooljet https://github.com/ToolJet/helm-charts.git
Error: looks like "https://github.com/ToolJet/helm-charts.git" is not a valid chart repository or cannot be reached: failed to fetch https://github.com/ToolJet/helm-charts.git/index.yaml : 404 Not Found
```

I took the new link from the readme of this repo https://github.com/ToolJet/helm-charts